packages/cli/src/workflows/workflowHistory/workflowHistory.service.ee.ts

```diff
@@ -66,13 +67,20 @@ export class WorkflowHistoryService {
 
 	async saveVersion(user: User, workflow: WorkflowEntity, workflowId: string) {
 		if (isWorkflowHistoryEnabled()) {
```

Comment:
Feels a bit odd to me to have this check here for enablement and the caller checking for licensed. Maybe it makes sense to put both in the same place, preferably inside the function.